core/version.h

```diff
@@ -64,3 +66,8 @@
 // Same as above, but prepended with Godot's name and a cosmetic "v" for "version".
 // Example: "Godot v3.1.4.stable.official.mono"
 #define VERSION_FULL_NAME "" VERSION_NAME " v" VERSION_FULL_BUILD
+
+// Godot's packed file magic header ("GDPC" in ASCII)
```

Comment:
`core/io/file_access_pack.h` might be a better place for these.

Reply:
I thought of putting them there at first, but from what I understand from the `file_access_*.cpp` family of functions the idea is that you don't depend on the specific `file_access_pack` header, but on the generic `file_access` header.

If we put the constants there then we have to include that header everywhere the code uses these constants, while `version.h` is already included. I thought that would negate the advantage of the `file_access` abstraction, thus I put them in `version.h`. Is it ok to directly depend on these headers, or wouldn't that generate a lot of confusion through double-inclusion?

Reply:
I think it should be fine in file_access_pack.h, as well as the pack magic, since in this case you will only need to include it from the .cpp of the editor code that exports the scenes